.github/CONTRIBUTING.md

```diff
@@ -1,39 +1,35 @@
 # Contributing to Discord4J
-Firstly, thanks for considering helping me out on this project! As stated in the readme, "No one is perfect at 
-programming and I am no exception." So I welcome any Pull Requests and Bug Reports. That being said, I have some 
-standards which I enforce in order to keep this project organized, maintainable and high-quality. 
+Firstly, we appreciate your interest in contributing to Discord4J!
+
+
+In order to maintain the high-quality standards of the project, we have a few rules to follow when contributing.
 
-## Issues vs Pull Requests
```

Comment:
The importance of issues should be more prominent as we should emphasize the need for issue reports to be tracked on GitHub as opposed to discord.

Reply:
Im not sure what "importance" you mean to emphasize. The templates are bug reports and feature requests, the only two things we want in the tracker.

Reply:
To emphasize that opening issues can be just as helpful as PRs.